pom.xml

```diff
@@ -199,7 +199,7 @@
       <dependency>
         <groupId>org.yaml</groupId>
         <artifactId>snakeyaml</artifactId>
-        <version>1.26</version>
+        <version>1.32</version>
```

Comment:
This is probably worth accepting, with a line in the CHANGELOG describing the change though...